packages/web/src/components/upload/EditableLabel.tsx

```diff
@@ -0,0 +1,54 @@
+import { ChangeEvent, useCallback } from 'react'
```

Comment:
Going to move this to harmony, probably as a follow-on, as i think there is some process for introducing new components to harmony